src/app/qgsrelationmanagerdialog.h

```diff
@@ -41,6 +41,7 @@ class APP_EXPORT QgsRelationManagerDialog : public QWidget, private Ui::QgsRelat
     void on_mBtnAddRelation_clicked();
     void on_mBtnDiscoverRelations_clicked();
     void on_mBtnRemoveRelation_clicked();
+    void onSelectionChanged();
```

Comment:
I know it's probably my fault ;) but while you're at it, could you make these slots private?